normandy/recipes/urls.py

```diff
@@ -7,12 +7,14 @@
     ActionViewSet,
     ClassifyClient,
     RecipeViewSet,
+    RecipeHistoryViewSet,
 )
 
 # API Router
 router = DefaultRouter()
 router.register(r'action', ActionViewSet)
 router.register(r'recipe', RecipeViewSet)
+router.register(r'recipe_history', RecipeHistoryViewSet)
```

Comment:
I think this should be called `recipe_version` to match the pattern of using the name of the thing that you're querying for.